Jenkinsfile

```diff
@@ -42,9 +42,8 @@ for (int i = 0; i < platforms.size(); ++i) {
                     junit '**/target/surefire-reports/TEST-*.xml'
 
                     if (label == 'linux') {
-                      archiveArtifacts artifacts: 'target/**/*.jar'
-                      // FindBugs is not enabled for this repo so far
-                      // findbugs pattern: '**/target/findbugsXml.xml'
+                      archiveArtifacts artifacts: '**/target/**/*.jar'
```

Comment:
Maybe just delete this. I do not think we need to archive artifacts here at all. The JAR files are not usable in isolation, only from the context of a source build.

Reply:
Why?

Reply:
You can copy a single file and then use it without "the context of a source build."
E.g. I do it in #53 for the Docker Image.

But I agree that other files can be probably excluded from publishing

Reply:
@oleg-nenashev ACK. My proposal was (years ago) to define a release process for this toolkit and avoiding its building each time.

Reply:
@recena Do you have a JIRA ticket for that? I think it is almost resolved:
* Since #54 this repo uses a standard Jenkins Plugin POM. To release the components you just need permissions to be configured in https://github.com/jenkins-infra/repository-permissions-updater/
* #53 introduced Docker packaging, which can be automated

Reply:
@oleg-nenashev I don't have but I can have it 😄 Although it would be needed to review some aspect of this toolkit: maintainers, next steps in short-term, etc...